pgjdbc/src/main/java/org/postgresql/jdbc/PgConnection.java

```diff
@@ -144,6 +144,8 @@
   private boolean readOnly = false;
   // Filter out database objects for which the current user has no privileges granted from the DatabaseMetaData
   private boolean  hideUnprivilegedObjects ;
+  // avoid generating server error detail if false;
+  private boolean logServerErrorDetail;
```

Comment:
This member variable can be `final`.

And for the comment how about: `// Whether to include error details in logging and exceptions`

Reply:
👍